inst/@sym/solve.m

```diff
@@ -112,48 +112,64 @@
   if (nargout == 0 || nargout == 1)
     cmd = { 'eqs = list(); symbols = list()'
             'for arg in _ins:'
-            '    if arg.is_Relational:'
-            '        eqs.append(arg)'
-            '    else:'
-            '        symbols.append(arg)'
+            ' if arg.is_Relational:'
+            '  eqs.append(arg)'
+            ' else:'
+            '  symbols.append(arg)'
```

Comment:
I don't think you changed these four lines...  This indenting was correct before your change: Python should use four spaces for indenting